nodes/src/nodes/llm_openai_api/openai_client.py

```
@@ -0,0 +1,78 @@
+
+        # Get the api key and base url
+        apikey = config.get('apikey')
+        base_url = config.get('base_url') or None
+
+        # Build kwargs for ChatOpenAI
+        kwargs = dict(
+            model=self._model,
+            api_key=apikey,
+            temperature=0,
+            max_tokens=self._modelOutputTokens,
+        )
```

Ruff (0.15.7)

[warning] Unnecessary `dict()` call (rewrite as a literal)

Rewrite as a literal

(C408)